lib/_layouts/_base.njk

```diff
@@ -3,88 +3,123 @@
 {% from "hero/macro.njk" import hero as appHero %}
 {% from "appFooter/macro.njk" import appFooter as oglFooter %}
 {% from 'back-link/macro.njk' import backLink %}
+{% from 'cookie-banner/macro.njk' import cookieBanner %}
 {% set htmlClasses = 'no-js' %}
 {% set homeUrl = '/' | url %}
 {% set containerClasses = containerClasses | default("app-width-container") %}
 <html lang="{{ htmlLang | default('en') }}">
 
-{% block head %}
-<head>
-  {% block headStart %}
+  {% block head %}
+    <head>
+      {% block headStart %}{% endblock %}
```

Comment:
@nhsbsa-Will , the diff on this has gone haywire. Can you revert this endblock back to a newline?